index.js

```diff
@@ -261,6 +262,41 @@ client.on("interactionCreate", async (interaction) => {
 
         await interaction.editReply(`You have sent ${messageCount} messages to ${user.username} since their last reply.`);
     }
+
+    // Check message for replies:
+    if (commandName === "checkreply") {
+        const messageId = options.getString("messageid");
+
+        try {
+            const channel = interaction.channel;
+            const originalMessageLink = `https://discord.com/channels/${interaction.guildId}/${channel.id}/${messageId}`;
+
+            // Check for replies:
+            const replies = [];
+            const fetchedMessages = await channel.messages.fetch({ limit: 100 });
```

Comment:
What if there was a reply, but it was more than 100 messages ago?

Reply:
This feature currently doesn't work for a message that was more than 100 messages ago. According to Discord.js documentation, 100 messages is the limit for each request.